spec/bolt/outputter/human_spec.rb

```diff
@@ -255,6 +255,39 @@
     expect(output.string).to match(/Finished on #{target}.*value/m)
   end
 
+  it "doesn't stacktrace when merged_output is nil" do
+    value = {
+      'stdout'        => 'stdout',
+      'stderr'        => 'stderr',
+      'merged_output' => nil,
+      'exit_code'     => 2
+    }
+    outputter.print_result(Bolt::Result.for_command(target, value, 'command', "executed", []))
+    expect { output.string }.not_to raise_error
+  end
+
+  it "matches standard out output" do
+    value = {
+      'stdout'        => 'stdout',
+      'stderr'        => nil,
+      'merged_output' => "stdout\nstderr",
+      'exit_code'     => 2
+    }
+    outputter.print_result(Bolt::Result.for_command(target, value, 'command', "executed", []))
+    expect(output.string).to match(/stdout/m)
+  end
+
+  it "matches standard error output" do
+    value = {
+      'stdout'        => nil,
+      'stderr'        => 'stderr',
+      'merged_output' => "stdout\nstderr",
+      'exit_code'     => 2
+    }
+    outputter.print_result(Bolt::Result.for_command(target, value, 'command', "executed", []))
+    expect(output.string).to match(/stderr/m)
+  end
```

Comment:
Because `merged_output` is set in both of these tests, the outputter will print what's in `merged_output` and not what's in `stdout`/`stderr`. While these tests both pass, they're not quite testing what we want: that the outputter falls back to printing `stdout`/`stderr` when `merged_output` is `nil`. The test on line 258 already prints a result like this, so we can either update that test to also check the output string or write a separate test that checks the output string.